components/UI/Map/Point/Point.tsx

```diff
@@ -41,10 +45,26 @@ function PointComponent({
             [styles.point_open]: isOpen,
             [styles.point_size_s]: size === Sizes.S,
             [styles.point_size_m]: size === Sizes.M,
+            [styles.point_noPreview]: !preview,
         });
 
-        return `<div class="${className}" style="${style}"></div>`;
-    }, [isOpen, size, color, sizeNumber]);
+        if (preview) {
+            return `<img
+                src="${preview}"
+                class="${className}"
+                style="${style}"
+                alt="Превью объекта"
```

Comment:
Let's remove the same `alt=""` and add more useful information. For example, with the name of the object.

Or until we delete it altogether.

Reply:
It is ideal to wrap everything in buttons with the `aria-haspopup` attribute

Reply:
let's do it in separate task, i have a couple of questions about roles